### ISSUE TITLE
Update Installation Documentation

### ISSUE THREAD

**Issue by emartinez-usgs** [usgs/earthquake-hazard-tool#107]
For both Windows and Mac. Start on a fresh machine and document each step that is required in order to get the application up and running.

Separate steps that will be common across all applications vs. specific to this application.

**Comment by jdbrown-USGS** [comment]
https://github.com/usgs/earthquake-hazard-tool/pull/118

**Comment by emartinez-usgs** [comment]
I'd like to try these out before merging. Looking good though. Waiting on laptop for testing...

**Comment by emartinez-usgs** [comment]
This stuff is a bit outdated now as we use other means for setting up the tools. Also, there are some concerns with the wording. Closing as won't fix since this isn't really an issue for us at this time.
